boards/adafruit/metro_esp32s3/doc/index.rst

```diff
@@ -0,0 +1,307 @@
+   - **Reset and DFU** (BOOT0) buttons to get into the ROM bootloader (which is a USB serial port so you don't need a separate cable!)
+   - **JTAG 2x5 Header** for more intense debugging
+   - **Serial debug output pins** (optional, for checking the hardware serial debug console)
+   - **STEMMA QT** connector for I2C devices, with switchable power, so you can go into low power mode
+   - **On/Charge/User** LEDs + status **NeoPixel** with pin-controlled power for low power usage
+   - **Low Power friendly!** In deep sleep mode we can get down to  approximately 100uA of current draw from the Lipoly connection. Quiescent current is from the power regulator, ESP32-S3 chip, and Lipoly monitor. Turn off the NeoPixel and external I2C power for the lowest quiescent current draw.
+
+**Processor features:**
+
+ESP32-S3 is a low-power MCU-based system on a chip (SoC) with integrated 2.4 GHz Wi-Fi
+and Bluetooth® Low Energy (Bluetooth LE). It consists of high-performance dual-core microprocessor
+(Xtensa® 32-bit LX7), a low power coprocessor, a Wi-Fi baseband, a Bluetooth LE baseband,
+RF module, and numerous peripherals.
+
+Digital interfaces:
+
+- 45 programmable GPIOs
+- 4x SPI
+- 1x LCD interface (8-bit ~16-bit parallel RGB, I8080 and MOTO6800), supporting conversion between RGB565, YUV422, YUV420 and YUV411
+- 1x DVP 8-bit ~16-bit camera interface
+- 3x UART
+- 2x I2C
+- 2x I2S
+- 1x RMT (TX/RX)
+- 1x pulse counter
+- LED PWM controller, up to 8 channels
+- 1x full-speed USB OTG
+- 1x USB Serial/JTAG controller
+- 2x MCPWM
+- 1x SDIO host controller with 2 slots
+- General DMA controller (GDMA), with 5 transmit channels and 5 receive channels
+- 1x TWAI® controller, compatible with ISO 11898-1 (CAN Specification 2.0)
+- Addressable RGB LED, driven by GPIO38.
+
+Analog interfaces:
+
+- 2x 12-bit SAR ADCs, up to 20 channels
+- 1x temperature sensor
+- 14x touch sensing IOs
+
+Timers:
+
+- 4x 54-bit general-purpose timers
+- 1x 52-bit system timer
+- 3x watchdog timers
+
+Low Power:
+
+- Power Management Unit with five power modes
+- Ultra-Low-Power (ULP) coprocessors: ULP-RISC-V and ULP-FSM
+
+Security:
+
+- Secure boot
+- Flash encryption
+- 4-Kbit OTP, up to 1792 bits for users
+- Cryptographic hardware acceleration: (AES-128/256, Hash, RSA, RNG, HMAC, Digital signature)
+
+Asymmetric Multiprocessing (AMP)
+********************************
+
+ESP32-S3 allows 2 different applications to be executed in the ESP32-S3 SoC. Due to its dual-core
+architecture, each core can be enabled to execute customized tasks in stand-alone mode
+and/or exchanging data over OpenAMP framework. See :ref:`ipc_samples` folder as code reference.
+
+For more information, check the datasheet at `ESP32-S3 Datasheet`_.
+
+Supported Features
+==================
+
+The current Zephyr implementation of the Adafruit Metro ESP32-S3 board supports the following features:
```

Comment:
this should have the board name as used by west build with \`\` around it and referred to as board target